### DISCUSSION TITLE
is there a list of badges somewhere?

### DISCUSSION THREAD

**Discussion by codespako** [community/community#30700] [Other Feature Feedback, Questions, & Ideas] [answered]
is there a list of badges somewhere??

**Comment by DenverCoder1** [comment]
Check out https://github.com/Schweinepriester/github-profile-achievements

**Comment by n1kkec** [comment, marked as answer]
First of all check for similar repositories or do a google search about it.

https://github.com/Schweinepriester/github-profile-achievements

But this is the link.

**Comment by codespako** [comment]
Thank you guys!
